packages/shared/src/validation/addressValidation.ts

```diff
@@ -26,36 +27,50 @@ export const validateBitcoinTestnetAddress: AddressValidator = (address: string)
 export const validateBitcoinRegtestAddress: AddressValidator = (address: string) =>
   bitcoin.isValidAddressForNetwork(address, 'regtest');
 
+export const validateSolanaAddress: AddressValidator = (address: string) => {
+  try {
+    base58.decode(address);
+    return true;
+  } catch {
+    return false;
+  }
+};
```

Comment:
can we move this to the `@chainflip/solana` package?
We will end up with 2 different validation functions (one more strict than the other), but i think they will both be useful and we might need the one that is less-strict (this one) on the frontends as well.